### PR TITLE
Add port-forward to ml-pipeline-ui service

### PR DESCRIPTION
**Pull request by diegolovison**
Add port-forward to ml-pipeline-ui service

### CODE REVIEW

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 1]
```diff
@@ -24,4 +24,11 @@ You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
      kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION"
      ```
 
-     The Kubeflow Pipelines deployment requires approximately 3 minutes to complete.
\ No newline at end of file
+     The Kubeflow Pipelines deployment requires approximately 3 minutes to complete.
+
+2. Run the following to port-forward:
```
Comment (hbelmiro) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:29]:
```suggestion
2. Run the following to port-forward the Kubeflow Pipelines UI:
```

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md [hunk 2]
```diff
@@ -24,4 +24,11 @@ You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
...
+     ```
+     kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+     ```
+
+3. Open [kubeflow-pipelines](http://localhost:8080)
```
Comment (hbelmiro) [inline content/en/docs/components/pipelines/operator-guides/installation/_index.md:34]: I think we can leave just the address here and leave the obvious explicit.

```suggestion
3. Open http://localhost:8080 on your browser to see the Kubeflow Pipelines UI.
```

Comment (franciscojavierarceo) [reply content/en/docs/components/pipelines/operator-guides/installation/_index.md:34]: +1

File: content/en/docs/components/pipelines/operator-guides/installation/_index.md (hunk 1 above)
Comment (franciscojavierarceo) [reply content/en/docs/components/pipelines/operator-guides/installation/_index.md:29]: +1

**Review by DharmitD — COMMENTED** [review]
/lgtm

**Review by hbelmiro — APPROVED** [review]
/lgtm

Thank you @diegolovison.

**Pull request merged**